.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,18 @@
+- id: sqlfluff-lint
+  name: sqlfluff-lint
+  entry: sqlfluff lint
+  language: python
+  description: 'Lints sql files with `sqlfluff`'
+  types: [sql]
+  require_serial: true
+  additional_dependencies: []
+
+- id: sqlfluff-fix
+  name: sqlfluff-fix
+  # Needs to use "--force" to disable confirmation
+  entry: sqlfluff fix --force
```

Comment:
In the latest version, can we just say `sqlfluff fix` and it will automatically apply all rules? I didn't know we had implemented that yet

Reply:
With #583 I think this ambiguity has been resolved

Reply:
I added a comment to mention that all the rules are applied by default